config/index.js

```diff
@@ -25,18 +34,28 @@ function printConfig (config) {
 *    ]
 *
 */
-module.exports = function (config, overrides) {
-  config.load = function () {
+module.exports = function ({ config, path: overridePath, overrides }) {
```

Comment:
@mdaum @Epitrochoid This may affect your repos if you're using this already (options object w/ destructuring instead of arg list)